### PR TITLE
Fix phpstan false positives

### PR DESCRIPTION
**Pull request by soyuka**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tickets       | na
| License       | MIT
| Doc PR        | na

False positives are reported on the path of the file

### CODE REVIEW

File: phpstan.neon.dist [hunk 1]
```diff
@@ -27,7 +27,7 @@ parameters:
 		# False positives
 		-
 			message: '#Variable \$positionPm might not be defined\.#'
-			path: %currentWorkingDirectory%/src/Util/ClassInfoTrait.php
+			path: %currentWorkingDirectory%
```
Comment (teohhanhui) [inline phpstan.neon.dist]: Woot. This looks bad...

**Comment by teohhanhui** [comment]
Actually, let's just exclude this version of phpstan... It's already fixed in https://github.com/phpstan/phpstan/commit/d0ae56e0e284a83f8dca55b48d7349d66cb6e604

File: phpstan.neon.dist (hunk 1 above)
Comment (soyuka) [inline phpstan.neon.dist]: or else I should specify every paths we use ClassInfoTrait, what do you prefer?

Comment (soyuka) [reply phpstan.neon.dist]: if you prefer:

```
(fix-phpstan ✗)❯ grep -Rli 'ClassInfoTrait' ./src
./src/Metadata/Property/Factory/SerializerPropertyMetadataFactory.php
./src/EventListener/WriteListener.php
./src/Util/ClassInfoTrait.php
./src/Util/ResourceClassInfoTrait.php
./src/Bridge/Doctrine/EventListener/PublishMercureUpdatesListener.php
./src/Bridge/Doctrine/Common/DataPersister.php
./src/Bridge/Symfony/Messenger/DataPersister.php
./src/Bridge/Symfony/Routing/IriConverter.php
./src/Serializer/AbstractItemNormalizer.php
./src/JsonApi/Serializer/ObjectNormalizer.php
./src/JsonApi/Serializer/ItemNormalizer.php
./src/GraphQl/Resolver/Factory/ItemResolverFactory.php
./src/GraphQl/Resolver/Factory/ItemMutationResolverFactory.php
./src/GraphQl/Resolver/Stage/ReadStage.php
./src/GraphQl/Resolver/ResourceFieldResolver.php
./src/GraphQl/Serializer/ObjectNormalizer.php
./src/GraphQl/Serializer/ItemNormalizer.php
./src/Api/CachedIdentifiersExtractor.php
./src/Api/ResourceClassResolver.php
./src/Api/IdentifiersExtractor.php
./src/JsonLd/Serializer/ItemNormalizer.php
./src/JsonLd/ContextBuilder.php
./src/Hal/Serializer/ItemNormalizer.php
``` 

:D

**Pull request merged**